Adyen/Service/Checkout.cs

```diff
@@ -67,6 +68,12 @@ public Checkout(Client client) : base(client)
         /// <returns>PaymentsResponse</returns>
         public PaymentResponse Payments(PaymentRequest paymentRequest, RequestOptions requestOptions = null)
         {
+            //if it is set by the merchant do not override
+            if (paymentRequest.ApplicationInfo == null)
```

Comment:
Do we want to allow users to change the application? This might skew any data we collect on library use. What are the use cases?